### PR TITLE
full Slack how-to update 

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

**Comment by stefaniebutland** [comment]
@jules32 Please request any edits to my new Slack documentation.

Now 
- Approach/Tooling#Slack describes how we use the tool
- Champions/Cohort tech setup#Slack describes setup for Champions, with example email
- Leading cohort calls where info used to live, now points to Cohort tech setup above

File: approach/tooling.md [hunk 1]
```diff
@@ -2,48 +2,74 @@
...
+-   communications and technical writing (RMarkdown, bookdown, blogdown, Quarto)
+-   [RStudio and GitHub setup instructions](https://openscapes.github.io/series/how-to/setup-rstudio-github.html)
 
-### Google Calendar
-
-  
 ### Slack
 
-- <https://slack.com/downloads>
+We use Slack as a way to build community by hosting a space for people to connect with each other around Openscapes-relevant topics. Members include people who have participated in a Champions Program, Pathways to Open Science, or other Openscapes program.
```
Comment (jules32) [inline approach/tooling.md:38]:
```suggestion
We use Slack as a way to build community by hosting a space for people to connect with each other around Openscapes-relevant topics. Members include people who have participated and mentored in a Champions Program, Pathways to Open Science, or other Openscapes programs.
```

File: approach/tooling.md [hunk 2]
```diff
@@ -2,48 +2,74 @@
...
-### Zoom
+#### Channel definitions
+
+These are meant to help people know where to post or ask questions. These definitions are listed in the Slack channel Topic and Description (because the Topic preview is visible in the channel, while the Description is not).
+
+When people join, they automatically land in #welcome #general #seaside-chats #events #cool-finds #shared-joy #random.
+
+**#welcome**: A place for people new to our community to introduce themselves and for the rest of us to make them feel welcome; the default channel that members cannot leave if they leave all other channels
+
+**#general**: Openscapes team posts Openscapes announcements
```
Comment (jules32) [inline approach/tooling.md:48]:
```suggestion
**#general**: General announcements from the Openscapes team and others
```

**Review by jules32 — COMMENTED** [review]
Hi Stef, this looks great, thanks for updating the text and org here. Just a few small edits. Thank you for your work on the Slack channel descriptions!

**Comment by jules32** [comment]
Feel free to merge after addressing these comments!

**Pull request merged**